by-language/javascript-prisma/README.md

```diff
@@ -0,0 +1,40 @@
+# Using CrateDB with Prisma
+
+
+## About
+
+[Prisma] is a next-generation Node.js and TypeScript ORM.
+
+
+## Details
+
+This example shows how to use [Prisma Client] in a **simple Node.js script**, to
+read and write data in a CrateDB database.
+
+The folder has been scaffolded using this command:
+```shell
+npx try-prisma@latest --template javascript/script --install npm --name . --path .
+```
+
+
+## Usage
+
+### Create the database
+
+Run the following command to submit the SQL DDL to the database. This will create
+database tables for the `User` and `Post` entities that are defined in
+[`prisma/schema.prisma`](./prisma/schema.prisma).
+```shell
+npx prisma migrate dev --name init
+```
```

Comment:
Prisma wants to run a `CREATE SCHEMA` command when using a connection string like `postgresql://crate@localhost:5432/mydb?schema=public`.

```shell
$ npx prisma migrate dev --name init
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "mydb", schema "public" at "localhost:5432"

Error: Schema engine error:
ERROR: line 1:8: no viable alternative at input 'CREATE SCHEMA'
```

Reply:
When using `postgresql://crate@localhost:5432/?schema=doc` as database connection string, it trips on the `CREATE DATABASE` statement.

```shell
Error: P3014

Prisma Migrate could not create the shadow database. Please make sure the database user has permission to create databases. Read more about the shadow database (and workarounds) at https://pris.ly/d/migrate-shadow

Original error:
ERROR: line 1:8: no viable alternative at input 'CREATE DATABASE'
   0: schema_core::state::DevDiagnostic
             at schema-engine/core/src/state.rs:270
```

Reply:
https://github.com/crate/crate/issues/14601